plugins/DuskVerb/tools/tuner/verify_preset_vs_anchor.py

```
@@ -124,27 +124,27 @@ def render(preset, vst3):
 def evaluate(preset, anchor_path, slug, vst3):
     """Returns dict with all metrics + per-gate verdicts."""
-    render(preset, vst3)
     dv_path = str(OUTPUT_DIR / f"{slug}_impulse.wav")
     try:
+        render(preset, vst3)
         dv = compute_metrics(dv_path)
         ref = compute_metrics(anchor_path)
     except Exception as e:
         return {'preset': preset, 'error': str(e)}
 
     dv_rms = rms_db(dv_path); ref_rms = rms_db(anchor_path)
```

Ruff (0.15.18)

[error] Multiple statements on one line (semicolon)

(E702)